crates/core/src/host/host_controller.rs

```diff
@@ -35,6 +35,14 @@ impl HostThreadpool {
                 rt.spawn_blocking(|| thread.run());
                 Ok(())
             })
+            .panic_handler(|_| {
+                // TODO: determine appropriate behavior here.
+                //       This version prints the panic info,
+                //       but doesn't abort.
+                //       I (pgoldman 2023-09-19) have no clue why adding an abort call here
+                //       causes the panic info to not be printed, but it does.
+                eprintln!("HostThreadpool panic handler!");
```

Comment:
Where did this change come from?

Reply:
This was to uncover the problem with a `panic` in bitcraft. I left this here to make the PR fail before we have all eyes and I don't know if we should keep it to avoid being dark about panics...